instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/utils/MiscUtils.kt

```diff
@@ -139,6 +139,10 @@ fun cleanStorageFiles() {
         .filesDir.deleteRecursively()
 }
 
+fun cleanGlobalAttributes() {
```

Comment:
I think method name differs a bit from the actual content - it removes only one particular attribute, not all of them.

Reply:
The idea is to update this method once we will add more global attributes. Unfortunately we do not have a `clearGlobalAttributes` method in our API and I really do not want to use reflection for this. We will have to keep this updated manually.